src/solver/optimisation/opt_alloc_probleme_a_optimiser.cpp

```diff
@@ -176,9 +176,9 @@ void OPT_AugmenterLaTailleDeLaMatriceDesContraintes(PROBLEME_ANTARES_A_RESOUDRE*
     ProblemeAResoudre->NombreDeTermesAllouesDansLaMatriceDesContraintes = NbTermes;
 }
 
-void OPT_LiberationMemoireDuProblemeAOptimiser(PROBLEME_HEBDO* problemeHebdo)
+void OPT_LiberationMemoireDuProblemeAOptimiser(PROBLEME_HEBDO& problemeHebdo)
 {
-    PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudre = problemeHebdo->ProblemeAResoudre;
+    PROBLEME_ANTARES_A_RESOUDRE* ProblemeAResoudre = problemeHebdo.ProblemeAResoudre;
```

Comment:
From my understanding, `ProblemeAResoudre` is now a `unique_ptr`(see #1549). I recommend removing function `OPT_LiberationMemoireDuProblemeAOptimiser`.